bpo-38101: Update devguide triaging keywords (#570)
python/devguide: triaging.rst

| easy          | Fixing the issue should not take longer than a day for     |
|               | someone new to contributing to Python to solve.            |
+---------------+------------------------------------------------------------+
| easy (C)      | Fixing the issue should not take longer than a day for     |

> |               | someone new contributing to Python, focused on C/C++.      |

Reviewer: We just use C, so the mention of C++ here is out of place.

Author: Done.

+---------------+------------------------------------------------------------+

> | security_issue| The issue would fit as, or is related as a security issue. |

Reviewer: This should clarify the relationship with the "security" issue type: Why do they both exist? What's the difference? When should one use this or the other, or both?

I don't know the answers to these; some research will likely be required.

Author: From what I see, it looks like "security" alone might have a security issue while "security_issue" is a definite issue that needs a fix, I don't believe you can use both since the later is an escalation of just "security".

+---------------+------------------------------------------------------------+

> | PEP 3121      | The issue fit as, or is related to the PEP 3123 module     |

Reviewer: The PEP number here is off, it should be 3121. Also, a few words about the subject of the PEP, and [a link to it](https://www.python.org/dev/peps/pep-3121/), would be very helpful.

| easy          | Fixing the issue should not take longer than a day for     |
|               | someone new to contributing to Python to solve.            |
+---------------+------------------------------------------------------------+
| easy (C)      | Fixing the issue should not take longer than a day for     |
|               | someone new contributing to Python, focused on C.          |
+---------------+------------------------------------------------------------+

> | security_issue| The issue would fit as, or is related as a security issue. |

Reviewer: ```suggestion
| security_issue| This is a security issue or is related to one.             |
```

> |               | The main difference from "security" is that this is a      |

Reviewer: "security" is out of context here, so maybe write something "The main difference from the "secutiry" issue type is..."

Author: Added both changes, I think also since there is no security keyword in the bug tracker, the main difference is that is a definite problem right? While the security type above it is more for reporting a possible issue.

| easy          | Fixing the issue should not take longer than a day for     |
|               | someone new to contributing to Python to solve.            |
+---------------+------------------------------------------------------------+
| easy (C)      | Fixing the issue should not take longer than a day for     |
|               | someone new contributing to Python, focused on C.          |
+---------------+------------------------------------------------------------+
| security_issue| The issue would fit as, or is related as a security issue. |
|               | The main difference from "security" is that this is a      |
|               | definite security problem that has to be dealt with.       |
+---------------+------------------------------------------------------------+
| PEP 3121      | The issue fit as, or is related to the PEP 3121 module     |
|               | Which is the Extension Module Initialization and           |
|               | Finalization, For More information:                        |

> |               | https://www.python.org/dev/peps/pep-3121/                  |

Reviewer: The wording and capitalization here are a bit clumsy. My suggestion:

"The issue is related to \`PEP 3121\`_: Extension Module Initialization and Finalization."

(The above assumes adding the link for PEP 3121 below it.)

Author: I have made the requested changes; please review again
